frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/newTicketMenu/newTicketMenu.component.tsx

```diff
@@ -48,9 +43,12 @@ export const NewTicketMenu = ({ onContainerOrFederationClick, TriggerButton, ...
 				<Label onClick={(e) => e.preventDefault()}>
 					<FormattedMessage id="ticketTable.newTicket.select" defaultMessage="Select a Federation or Container" />
 				</Label>
-				{sortByName(selectableModels).map(({ _id, name }) => (
+				{sortByName(selectableModels).map(({ _id, name, role }) => (
 					<ActionMenuItem key={_id}>
-						<MenuItem onClick={() => onContainerOrFederationClick(_id)}>
+						<MenuItem
```

Comment:
This only works to stop a 'viewer' from creating an issue if there are multiple models selected. If there is just one model selected then this menu is bypassed and a viewer can start creating a ticket. But this is actually probably just due to the other problem listed below